spec.bs

```diff
@@ -3003,6 +3014,41 @@ The <dfn for=Navigator method>getInterestGroupAdAuctionData(|config|)</dfn> meth
 
 </div>
 
+<div algorithm>
+  To <dfn>parse and verify ad auction config</dfn> given an {{AdAuctionDataConfig}} |configIDL|:
+
+  1. Let |config| be a new [=auction data config=] with the following [=struct/items=]:
+    : [=auction data config/seller=]
+    :: The result of running [=parsing an https origin=] on
+       |configIDL|["{{AdAuctionDataConfig/seller}}"]
+    : [=auction data config/coordinator=]
+    :: The result of running [=parsing an https origin=] on
+       |configIDL|["{{AdAuctionDataConfig/coordinatorOrigin}}"]
+    : [=auction data config/request size=]
+    :: |configIDL|["{{AdAuctionDataConfig/requestSize}}"]
+    : [=auction data config/per buyer config=]
+    :: The result of running [=parse per buyer auction data configs=] on
+      |configIDL|["{{AdAuctionDataConfig/perBuyerConfig}}"]
+  1. If |config|'s [=auction data config/seller=] or [=auction data config/coordinator=]

```

Comment:
This seems to be missing the must-have-default-size check if no overall size is specified:
https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/renderer/modules/ad_auction/navigator_auction.cc;drc=7578f90a7173017ec95ded2bb21419ab14e57a16;l=4450 ?

Also am I right in reading 
https://source.chromium.org/chromium/chromium/src/+/main:content/browser/interest_group/interest_group_manager_impl.cc;drc=8e948282d37c0e119e3102236878d6f4d5052c16;l=716

as, if perBuyerConfig exists here, it controls which IGs to actually include?

Reply:
Good point on the missing size check. I've added that to the spec here.

Yes, if perBuyerConfig exists it controls which IGs to include. Currently that was planned to be in the IETF spec which filters them, but I could add a note here.

Reply:
Changed to add filtering here. It doesn't hurt to do it in both places.